### PR TITLE
Gimme-Cloud-Talks-Community-Addition

### PR DESCRIPTION
**Pull request by dwnatwick**
Information for Gimme Cloud Talks community to participate in Global Azure event.

### CODE REVIEW

**Comment by rikhepworth** [comment]
Hi. Your Sessionize feed is set to embed rather than json. Whilst we haven't enabled pulling agenda from communities onto our site yet, this will prevent your sessions appearing.

File: 2023/gimmecloudtalks/data.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
```
Comment (rikhepworth) [inline 2023/gimmecloudtalks/data.json:5]: We need a GitHub handle in here - for you that would be dwnatwick
e.g. 
"GitHubHandles": [ "dwnatwick" ]

**Comment by rikhepworth** [comment]
The index.md file is missing.
We need an index.md file because we use it to provide the information about your community that is displayed on our site. This could be as simple as a title and a link to your community site, or you could add more information about your community.

**Comment by dwnatwick** [comment]
Handles have been added and index.md file has been created.  Pull request should be ready now.  Thank you.

**Pull request merged**